profile.js

```diff
@@ -35,4 +35,64 @@ document.getElementById('menu').addEventListener('click', () => {
   document.getElementById('link1').addEventListener('click', hideMobMenu);
   document.getElementById('link2').addEventListener('click', hideMobMenu);
   document.getElementById('link3').addEventListener('click', hideMobMenu);
-});
\ No newline at end of file
+});
+// || =========== Pop-Up Modal ============ ||
+const arrProjects = [];
+function addProjects() {
+  // Stores Project Infos in arrProject
+  const colTitles = document.getElementsByClassName('project-titles');
+  const colClient = document.getElementsByClassName('client');
+  const colRole = document.getElementsByClassName('type');
+  const colYear = document.getElementsByClassName('year');
+  for (let i = 0; i < colTitles.length; i += 1) {
+    // | creating a new project object an pushing it to arrProject
+    const objProject = {};
+    objProject.name = colTitles[i].textContent;
+    objProject.client = colClient[i].textContent;
+    objProject.role = colRole[i].textContent;
+    objProject.year = colYear[i].textContent;
+    objProject.tags = ['html', 'css', 'javascript', 'github', 'ruby', 'bootstrap'];
+    objProject.snapshot = `images/SnapshootD${i + 1}.jpg`;
+    arrProjects.push(objProject);
+  }
+}
+
+function getProjectDetails(proID) {
+  // Retrieving  Project Info
+  return arrProjects[proID];
+}
+
+function showProjectDetails(proID) {
+  // showing  Project Info
+  const modPopUp = document.getElementById('modal-ui-wrapper');
+  const p = getProjectDetails(proID - 1);
+  document.getElementById('mc-title').textContent = p.name;
+  document.getElementById('mc-client').textContent = p.client;
```

Comment:
- [ ] You did a very good job on the styles but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1143) design as much as possible. Please increase the width of the model and top-bottom margins and make sure it matches the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/169248295-c1bb2c86-eb23-4916-ab85-8e6ebe513273.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/169248204-0b257ca1-1c6f-47f5-b5a0-8c89a5ce5e65.png)